src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -33,16 +33,22 @@ public enum IamAction {
   UPDATE_BILLING_ACCOUNT,
   LINK;
 
+  private final String samActionName;
+
+  IamAction() {
+    this.samActionName = StringUtils.lowerCase(name());
```

Comment:
Since `name` can't be `null` there's no hazard with calling the `String` API directly
```suggestion
    this.samActionName = name().toLowerCase();
```